tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/xml/XMLAttributeAccessTest.java

```diff
@@ -104,6 +104,11 @@ public void testXMLAfterRemoveAttribute() {
         BRunUtil.invoke(compileResult, "testXMLAfterRemoveAttribute");
     }
 
+    @Test
+    public void testXMLIndexedOptionalAttributeAccess() {
```

Comment:
Given the function `testXMLIndexedOptionalAttributeAccess` could return an error shouldn't we verify that it didn't here? Otherwise if any of the checked expressions return an error this test could silently fail. Perhaps a better option may be to do the "not an error" test in ballerina side as well by adding a function like `assertNotError`